elements/rh-alert/rh-alert.ts

```diff
@@ -54,6 +63,11 @@ export class RhAlert extends LitElement {
 
   @property({ reflect: true, type: Boolean }) toast = false;
 
+  @bound private _closeHandler() {
```

Comment:
```suggestion
  #closeHandler() {
```
no need for the decorator if it's attached in the template